### ISSUE TITLE
Documentation: ELECTRA Pretraining

### ISSUE THREAD

**Issue by stefan-it** [tensorflow/models#11054]
Dear TF Models Garden Maintainer,

today I recognized that TensorFlow 1.15 is no longer available as supported TPU version. That means, it is no longer possible to pretrain [BERT](https://github.com/google-research/bert) and [ELECTRA](https://github.com/google-research/electra) models with the official code bases.

As I was able to sucessfully pretrain Token Dropping BERT and TEAMS models (many thanks for open sourcing the implementation), I would like to know if it's already possible to pretrain "normal" ELECTRA models with TF Models Garden and if the necessary pretraining commands can be provided here and in the documentation.

Many thanks in advance!

**Comment by stefan-it** [comment]
Update: I could find a section about BERT pretraining a while ago in this commit:

https://github.com/tensorflow/models/blob/bc8e6d24c21ed2939025a066095f3cab198547db/official/nlp/docs/train.md

But it is no longer availble in latest `master` version.

**Comment by laxmareddyp** [comment]
Hi @Chen-Zhe,

Could you please check this in your free time.

Thanks

**Comment by malla456** [comment]
Hi @stefan-it ,


Pretraining ELECTRA models is fully supported in TensorFlow 2.x using the official NLP training library. You can start the pretraining process by running the official/nlp/train.py script with the --experiment=electra/pretraining flag. The model size and hyperparameters are easily controlled by passing the appropriate YAML configuration file to the --config_file argument. Furthermore, this implementation remains fully compatible with existing TF 1.x pretraining data while offering a new create_pretraining_data.py script for generating datasets with advanced masking features.
